dev/TEAL_opcodes.md

```diff
@@ -1346,10 +1360,39 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - Opcode: 0xb5 {uint8 transaction field index} {uint8 transaction field array index}
 - Pops: _None_
 - Pushes: any
-- push Ith value of the array field F of the last inner transaction to stack
+- push Ith value of the array field F of the last inner transaction
 - LogicSigVersion >= 5
 - Mode: Application
 
+## itxn_next
+
+- Opcode: 0xb6
+- Pops: _None_
+- Pushes: _None_
+- begin preparation of a new inner transaction in the same transaction group
+- LogicSigVersion >= 6
+- Mode: Application
+
+`itxn_next` initializes the transaction exactly as `itxn_begin` does
+
+## gitxn t f
+
+- Opcode: 0xb7 {uint8 transaction group index} {uint8 transaction field index}
+- Pops: _None_
+- Pushes: any
+- push field F of the Tth transaction in the last inner group
```

Comment:
What is the last inner group? The last one that was `itxn_submit`ted, or the one that had `itxn_begin` but not `itxn_submit`ted?

Reply:
The last one submitted.  You can not examine the itxns that you are building. (Seemed both pointless, and a potential place where people might exploit it for extra temporary space.). I'll add a clarifying word or two.